Sources/NIO/EventLoopFuture.swift

```diff
@@ -681,9 +683,9 @@ extension EventLoopFuture {
     /// provided `EventLoopFuture` both succeed. It then provides the pair
     /// of results. If either one fails, the combined `EventLoopFuture` will fail with
     /// the first error encountered.
-    public func and<U>(_ other: EventLoopFuture<U>, file: StaticString = #file, line: UInt = #line) -> EventLoopFuture<(T,U)> {
-        let promise = EventLoopPromise<(T,U)>(eventLoop: eventLoop, file: file, line: line)
-        var tvalue: T?
+    public func and<U>(_ other: EventLoopFuture<U>, file: StaticString = #file, line: UInt = #line) -> EventLoopFuture<(Value, U)> {
```

Comment:
Same question here about `U`.

Reply:
thanks, renamed to `NewValue`, `OtherValue`, or `InputValue` depending on the context.